src/main/java/com/asem/regestration/model/dao/impl/StudentDAOIMpl.java

```diff
@@ -80,26 +72,32 @@ public Student findByEmail(String email) throws SQLException {
         return student;
     }
 
-
-
-
-
     @Override
-    public void deletStudent(String id) {
+    public void deleteStudent(String id) throws SQLException {
         String sql="delete from student2.student where student.id="+id+"";
```

Comment:
Use String.format instead of concatenating string
What does the last "" after id+ do ?